docs/DXIL.rst

```diff
@@ -515,33 +515,52 @@ Additional shader properties are specified via tag-value pair list, which is the
 Shader Flags
 ------------
 
-Shaders have additional flags that covey their capabilities via tag-value pair with tag kDxilShaderFlagsTag (0), followed by an i64 bitmask integer. The bits have the following meaning:
-
-=== =====================================================================
-Bit Description
-=== =====================================================================
-0   Disable shader optimizations
-1   Disable math refactoring
-2   Shader uses doubles
-3   Force early depth stencil
-4   Enable raw and structured buffers
-5   Shader uses min-precision, expressed as half and i16
-6   Shader uses double extension intrinsics
-7   Shader uses MSAD
-8   All resources must be bound for the duration of shader execution
-9   Enable view port and RT array index from any stage feeding rasterizer
-10  Shader uses inner coverage
-11  Shader uses stencil
-12  Shader uses intrinsics that access tiled resources
-13  Shader uses relaxed typed UAV load formats
-14  Shader uses Level9 comparison filtering
-15  Shader uses up to 64 UAVs
-16  Shader uses UAVs
-17  Shader uses CS4 raw and structured buffers
-18  Shader uses Rasterizer Ordered Views
-19  Shader uses wave intrinsics
-20  Shader uses int64 instructions
-=== =====================================================================
+Shaders have additional flags that convey their capabilities via a tag-value pair with tag kDxilShaderFlagsTag (0), followed by an i64 bitmask integer. The bits have the following meaning, Shader Model requirements, and criteria for being set:
+
+=== ==== ======================================================================================== ==========================================================================================================================================================================================================================================================================================================================================================
+Bit SM   Description                                                                              Criteria to set shader flag
+=== ==== ======================================================================================== ==========================================================================================================================================================================================================================================================================================================================================================
+0        Disable shader optimizations                                                             Command-line flag ``-disable-llvm-optzns`` provided to DXC's internal frontend
```

Comment:
`-dixable-llvm-optzns` isn't really supported by DXC, I believe the flag is `-Od`.